go/vt/vtgate/vtgate.go

```diff
@@ -304,6 +306,7 @@ func Init(
 		si,
 		noScatter,
 		pv,
+		truncateErrorLen,
```

Comment:
`truncateErrorLen` does not need to be part of the parameter. `vtgate` and `executor` both have access to this field.

Reply:
good catch; updated!